python_orocos_kdl/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.4.6)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
I think we can skip this one; it is `AMENT_IGNORE`, and I'm certain we'd have to do more complicated things to make it work anyway.

Reply:
Realized you meant this change, not the PR in total. Removed this specific change